src/Http/Controllers/FilesController.php

```diff
@@ -93,6 +93,13 @@ private function files()
                     $data['fileMeta'][$key] = $file[$column] ?? null;
                 }
 
+                if ($this->tool->relationship === 'HasMany' || $this->tool->relationship === 'HasMany') {
```

Comment:
Duplicated  `$this->tool->relationship === 'HasMany'` maybe also HasOne can be useful so will be `if ($this->tool->relationship)`